src/Strings.js

```diff
@@ -0,0 +1,113 @@
+module.exports = {
```

Comment:
Then here, we'd have a function instead of an object: 

```js
module.exports = function(sequencer) {
```

Reply:
Or actually perhaps we need to pass in only `addSteps()` and `modulesInfo()` -- see below...